server.js

```diff
@@ -21,9 +23,55 @@ app.use(express.json());
 
 // Start defining your routes here
 app.get("/", (req, res) => {
-  res.send("Hello Technigo!");
+  // res.send({responseMessage: "Hello Technigo!"}); // we are send when we would like to sent translate json object
+  res.json ({responseMessage: "Hello Technigo!"}) // it is sending as a json, which makes it more secure if there is some error
```

Comment:
To make it easier for those who want to test or use your api you can add the different endpoints in this route :D